.github/workflows/accessibility.yml

```diff
@@ -0,0 +1,19 @@
+name: a11y
+
+on:
+  pull_request:
+  push:
+    branches: 
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: a11ywatch/github-action@v1.2.44
```

Comment:
```suggestion
      - uses: a11ywatch/github-action@v1.2.43
```

Reply:
44 doesn't exist yet, i opened a PR on their side to fix their docs

Reply:
Yep, just saw the same and pushed a new commit for this PR!